src/nomad_parser_vasp/schema_packages/vasp_package.py

```diff
@@ -43,7 +36,7 @@
 Simulation.outputs.m_annotations['xml'] = MappingAnnotationModel(path=".calculation")
 
 Program.name.m_annotations['xml'] = MappingAnnotationModel(
-    path="modeling.generator.i[?name == 'program']"
+    path='modeling.generator.i[?"@name"==\'program\'] | [0].__value',
```

Comment:
@ladinesa what does the `[0].__value` option cover?

Reply:
this is sort of standard i put up in the xml parser for the case where attributes are defined for a quantity so the quantity value itself is given by __value while the attribute keys are prefixed by `@`, you can modify these when you instantiate XMLParser

Reply:
the index [0] since jmespath will return a list when querying a list

Reply:
Thx for the clarification. Is the `@` consistent with the node definition in JMesPath?

Reply:
it has nothing to do with it i think, i am not even familiar with the usage of @ in jmespath tbh. we can change the default probably to some other token if @ is used in jmespath

Reply:
`@` means the current node: https://jmespath.org/specification.html#spec

Reply:
ah so, we can use it in place of '.' in my convention?nice to know.

Reply:
> this is sort of standard

If this should always or frequently be used, shouldn't it be moved to the XML parser?

Reply:
you can perhaps define this as a variable in parsers common utils but definitely not in nomad.